vercel.json

```diff
@@ -633,6 +633,26 @@
       "source": "/docs/start-building/native-mobile-app",
       "destination": "https://www.ory.sh/docs/getting-started/integrate-auth/react-native",
       "permanent": false
+    },
+    {
+      "source": "docs/kratos/concepts/identity-schema",
+      "destination": "https://www.ory.sh/docs/guides/manage-identities/overview",
+      "permanent": false
+    },
+    {
+      "source": "docs/guides/manage-identities/managing-users-identities",
+      "destination": "https://www.ory.sh/docs/guides/manage-identities/create-identities",
```

Comment:
Same here as in the SEO comment below. What would be a good google search term that we would want to rank this site for prominently?

Reply:
See the comment above, please